software/wmla120.py

```diff
@@ -260,23 +258,23 @@ def __del__(self):
         if not os.path.exists(GEN_SOFTWARE_PATH):
             os.mkdir(GEN_SOFTWARE_PATH)
         if self.eval_ver:
-            with open(GEN_SOFTWARE_PATH + 'software-vars-eval.yml', 'w') as f:
+            with open(GEN_SOFTWARE_PATH + f'{self.sw_vars_file_name}', 'w') as f:
```

Comment:
The if and else appear to be doing the same thing now that self.sw_vars_file_name has the name of the file.

Reply:
agree, i did not want to change too much since it was built that way.